pkgpanda/build/__init__.py

```diff
@@ -778,8 +778,18 @@ def get_build_ids(self):
 
 
 def build(package_store, name, variant, clean_after_build, recursive=False):
+    variant_str = pkgpanda.util.variant_str(variant)
+    if variant_str:
+        msg = "Building package {} variant {}".format(name, variant_str)
+    else:
+        msg = "Building package {}".format(name)
+    with logger.scope(msg):
+        print(msg)
```

Comment:
Why do we need both the logger and a print here? Sometimes we do both, sometimes we only do one. From my perspective shouldn't also need to print if I'm using the logger.